wagtail/contrib/forms/templates/wagtailforms/index_submissions.html

```diff
@@ -81,47 +81,38 @@
-                        </h1>
-                    </div>
-                    <div class="col search-bar">
-                        <ul class="fields row rowflush">
-                            {% for field in select_date_form %}
-                                {% include "wagtailadmin/shared/field_as_li.html" with field=field field_classes="field-small" li_classes="col4" %}
-                            {% endfor %}
-                            <li class="submit col2">
-                                <button type="submit" name="action" value="filter" class="button button-filter">{% trans 'Filter' %}</button>
-                            </li>
-                        </ul>
-                    </div>
-                </div>
-                <div class="right">
-                    <div class="dropdown dropdown-button match-width">
-                        <a href="?export=xlsx" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
-                        <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
-                        <ul>
-                            <li><a  class="button bicolor button--icon" href="?export=csv">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
-                        </ul>
-                    </div>
-                </div>
+    {% fragment as form_actions %}
+        <div class="dropdown dropdown-button">
+            <a href="?export=xlsx" class="button bicolor button--icon">{% icon name="download" wrapped=1 %}{% trans 'Download XLSX' %}</a>
+            <div class="dropdown-toggle">{% icon name="arrow-down" %}</div>
+            <ul>
+                <li><a  class="button bicolor button--icon" href="?export=csv">{% icon name="download" wrapped=1 %}{% trans 'Download CSV' %}</a></li>
+            </ul>
+        </div>
+    {% endfragment %}
+
+    {% include "wagtailadmin/shared/header.html" with title="Form data" subtitle=form_page.title|capfirst icon="form" merged=1 extra_actions=form_actions %}
+    <div class="form">
+        <div class="report report--has-filters nice-padding">
+            <div class="report__results w-overflow-y-hidden">
+                {% if submissions %}
+                    <form action="{% url 'wagtailforms:delete_submissions' form_page.id %}" method="get">
+                        {% include "wagtailforms/list_submissions.html" %}
+                        {% include "wagtailadmin/shared/pagination_nav.html" with items=page_obj %}
+                    </form>
+                {% else %}
+                    <p class="no-results-message">{% blocktrans trimmed with title=form_page.title %}There have been no submissions of the '{{ title }}' form.{% endblocktrans %}</p>
+                {% endif %}
+            </div>
+            <div class="report__filters">
```

Comment:
```suggestion
            <div class="report__filters w-p-3">
```

Adding padding here will help the content be inset without needing the styling on the outer div.form.